btf/traversal.go

```diff
@@ -86,6 +102,20 @@ func children(typ Type, yield func(child *Type) bool) bool {
 		if !yield(&v.Type) {
 			return false
 		}
+		tags = v.Tags
```

Comment:
Move this to the end of the `case` clause for consistency? I was wondering why we were ignoring v.Tags for Func. :stuck_out_tongue: